Sources/TuistKit/Services/GenerateService.swift

```diff
@@ -24,7 +24,7 @@ final class GenerateService {
         generatorFactory: GeneratorFactorying,
         clock: Clock = WallClock(),
         timeTakenLoggerFormatter: TimeTakenLoggerFormatting = TimeTakenLoggerFormatter(),
-        manifestLoader: ManifestLoading = ManifestLoader(),
+        manifestLoader: ManifestLoading = CachedManifestLoader(),
```

Comment:
I looked at the `GeneratorService` file in more detail and noticed that this variable is not used anywhere at all))

The manifest loader is taken from `generatorFactory`. And there it returns `CachedManifestLoader` by default.

At the same time, the change in `DumpService` is advisable.